lib/GADS.pm

```diff
@@ -4356,6 +4356,7 @@ sub _process_edit
 
     if (my $delete_id = param 'delete')
     {
+        # say STDERR 'delete';
         if (process( sub { $record->delete_current }))
```

Comment:
There are a few debug statements - should these be removed?

Reply:
Yes, these should be removed. I will include them in the next update